### PR TITLE
Adds Management Cluster field for Hypershift Clusters describe

### PR DESCRIPTION
**Pull request by fahlmant**
Adds the `Management Cluster` field to `ocm describe cluster` for Hypershift Hosted Control Plane clusters. This is the equivalent of `Shard` for classic OSD/ROSA.

### CODE REVIEW

File: pkg/cluster/describe.go [hunk 1]
```diff
@@ -157,6 +157,21 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
...
+	mgmtCluster := ""
+	if cluster.Hypershift().Enabled() {
+		hypershiftResp, err := connection.ClustersMgmt().V1().Clusters().
+			Cluster(cluster.ID()).
+			Hypershift().
+			Get().
+			Send()
+		if err != nil {
+			fmt.Println(err)
+			return err
```
Comment (mjlshen) [inline pkg/cluster/describe.go:170]:
```suggestion
				return fmt.Errorf(
					"can't get management cluster for '%s': %v",
					cluster.ID(), err,
				)
```

**Review by mjlshen — CHANGES_REQUESTED** [review]
We should return errors up instead of printing them

**Review by mjlshen — COMMENTED** [review]
/lgtm

**Review by rafael-azevedo — COMMENTED** [review]
/lgtm !

File: pkg/cluster/describe.go [hunk 2]
```diff
@@ -157,6 +157,20 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
...
+	// Check Hypershift-related values
+	mgmtCluster := ""
+	if cluster.Hypershift().Enabled() {
+		hypershiftResp, err := connection.ClustersMgmt().V1().Clusters().
+			Cluster(cluster.ID()).
+			Hypershift().
+			Get().
+			Send()
+		if err != nil {
+			return fmt.Errorf("unable to query hypershift endpoint for cluster %s: %v", cluster.ID(), err)
```
Comment (vkareh) [inline pkg/cluster/describe.go:169]: This error should be silently ignored as there are only a handful of roles that can access this endpoint.

**Review by vkareh — APPROVED** [review]
/lgtm

**Pull request merged**